src/StackAdmin/AzureStackStorage/Commands.AzureStackStorage.Tests/Commands.AzureStackStorage.Test.Netcore.csproj

```diff
@@ -1,53 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@deathly809 why are we deleting this file?

Reply:
Module no longer exists.